client/wc-api/notices/operations.js

```diff
@@ -0,0 +1,25 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { uniqueId } from 'lodash';
+
+function create( resourceNames, data ) {
+	return [ ...createNotice( resourceNames, data ) ];
+}
+
+function createNotice( resourceNames, data ) {
+	const resourceName = 'notices';
+
+	if ( resourceNames.includes( resourceName ) ) {
+		const notice = { ...{ id: uniqueId( 'notice-' ) }, ...data[ resourceName ] };
+		// @TODO: this should append the notice to the existing state.
+		return [ { [ 'notices' ]: { data: [ notice ] } } ];
```

Comment:
This is currently not a great implementation as it overwrites anything already existing in the store instead of appending a new one.  

Is there any way to grab an existing copy of the store to merge the new notice in or should we be creating an entirely new store instead of trying to fit this into fresh data? /cc @coderkevin